packages/terra-form-field/src/Field.scss

```diff
@@ -0,0 +1,84 @@
+@import '~terra-mixins';
+@import './cerner-mock-theme';
+
+:local {
+  .field {
+    border: 0;
+    margin-bottom: var(--terra-form-field-margin-bottom, 1.428em);
+    max-width: 1020px;
```

Comment:
This seems oddly specific, is there an animation or overflow this is enabling or is this a hard cutoff for field widths?

Reply:
This is the hard cutoff with widths. This value was 
> derived from Design Standard max of 60 characters  using default size 14pt all capital W’s.

Reply:
We should set this value in rems so it scales if the user adjusts the default font-size. With 14px being our computed root font size, the rem value for 1020px is `72.857rem`

Reply:
Reminder about updating this to `72.857rem`